### PR TITLE
Correct FipsUtils and add unit tests

### PR DESCRIPTION
**Pull request by rlcundiff**
https://github.com/OpenLiberty/open-liberty/issues/27649

### CODE REVIEW

**Comment by rlcundiff** [comment]
#build

File: dev/com.ibm.ws.crypto.common/test/com/ibm/ws/crypto/common/FipsUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package com.ibm.ws.crypto.common;
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.crypto.common/test/com/ibm/ws/crypto/common/FipsUtilsTest.java:10]:
```suggestion
/*******************************************************************************
 * Copyright (c) 2024 IBM Corporation and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License 2.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-2.0/
 * 
 * SPDX-License-Identifier: EPL-2.0
 *******************************************************************************/
package com.ibm.ws.crypto.common;
```

**Review by Zech-Hein — CHANGES_REQUESTED** [review]
the changes look good 👍 we just need to add the copyright header to the unit test class you added

**Review by Zech-Hein — APPROVED** [review]
LGTM

**Comment by rlcundiff** [comment]
#libby

**Comment by rlcundiff** [comment]
This is a new project that is still unused.  The build had a UI test failure that is unrelated.

**Pull request merged**